packages/ui/src/components/ChatMessage/AuthorRow.tsx

```diff
@@ -0,0 +1,45 @@
+import { utils } from '@tloncorp/shared';
+import * as client from '@tloncorp/shared/dist/client';
+import { SizableText, View, XStack } from 'tamagui';
+
+import { Avatar } from '../Avatar';
+import ShipName from '../ShipName';
+
+const RoleBadge = ({ role }: { role: string }) => {
+  return (
+    <View
+      borderRadius="$l"
```

Comment:
Probably not there yet, but I'm curious if there are wrapper types it makes sense to extract.

Reply:
Not sure what you mean on this one.

Reply:
Haha yeah, not clear -- just wondering out loud if we'll want any `PaddedBox` type components.